torch/functional.py

```diff
@@ -263,3 +263,46 @@ def det(var):
     if torch.is_tensor(var):
         raise ValueError("det is currently only supported on Variable")
     return var.det()
+
+
+def stft(var, frame_length, hop, window=None, pad_end=0):
+    r"""stft(signal, frame_length, hop, window=None, pad_end=0) -> Tensor, Tensor
```

Comment:
Generally, you don't rewrite the signature for Python functions. https://www.python.org/dev/peps/pep-0257/